SU-SOE/soe_basic
# D8CORE-3247: Publications styles

# READY FOR REVIEW

# Summary
- Adding classes and removing the title on the publication pages.

# Needed By (Date)
- 3/3

# Urgency
- high

# Steps to Test

1. Pull in this change.
2. Pull in the SOE profile change: https://github.com/SU-SOE/soe_profile/pull/108
3. Refresh everything
4. Make some publications content.
5. Verify that publications has the gradient header on the /publications and the /publications/term name
6. Look at the publications in a list

# Affected Projects or Products
- SOE

# Associated Issues and/or People
- D8CORE-3577
 - D8CORE-3247
 - D8CORE-3580
- https://github.com/SU-SOE/soe_basic/pull/66
